bor/client/rest/query.go

```diff
@@ -245,7 +287,7 @@ func prepareNextSpanHandlerFn(cliCtx context.CLIContext) http.HandlerFunc {
 		msg := hmTypes.NewSpan(
 			spanID,
 			startBlock,
-			startBlock+spanDuration-1,
+			startBlock+spanDuration,
```

Comment:
```suggestion
			startBlock + spanDuration - 1,
```

Example: Let's span duration is `6400` and `startBlock` is `0`. Then `endBlock` should be:

```
endBlock := startBlock + spanDuration - 1 = 0 + 6400 - 1 = 6399
```

So, now span will start at block `0` and end at `6399`, total of `6400`